etl/steps/data/garden/worldbank_wdi/2024-05-20/wdi.meta.yml

```diff
@@ -2544,6 +2547,7 @@ tables:
           name: GDP per capita
           numDecimalPlaces: 0
         presentation:
+          title_public: GDP per capita, PPP (current international $)
```

Comment:
For the GDP indicators you could have:
title_public: GDP per capita
title_variant: In current international $
maybe @paarriagadap has better suggestions for these ones.